models/organization/org.go

```diff
@@ -90,6 +90,11 @@ func OrgFromUser(user *user_model.User) *Organization {
 	return (*Organization)(user)
 }
 
+// UserFromOrg converts organization to user
+func UserFromOrg(org *Organization) *user_model.User {
+	return (*user_model.User)(org)
```

Comment:
```suggestion
	return org.(*user_model.User)
```
Sounds more Go-y to me…

Reply:
`(*user_model.User)(org)` is right. Because `Org` is a type alias.